content/english/blog/post-oscar-text-to-speech.md

```diff
@@ -12,116 +12,142 @@ draft: false
-  * The code, implemented Python, uses the [Google Speech](https://pypi.org/project/google-speech/) library to convert text to speech.
-  * The user-script that will be executed inside the container when the service is triggered is in the file "script.sh" and it will start the Python program.
-  * Last but not least, the YAML-based Functions Definition File (FDL) provides the definition to create the service via oscar-cli.
+Coqui only uses the command line:
 
+- The Dockerfile is a [base image](https://github.com/orgs/grycap/packages/container/package/text-to-speech-coqui) based on Ubuntu 20.04 that installs all the dependencies.
+- In this case, the bash script runs commands. It does not execute a program that has been created exclusive storage in the docker images.
+- YAML-file to deploy the services
 
 #### STEP 1.1: Get the YAML file ready
 
-Check in the yaml file that the cluster identifier is defined (OSCAR-CLI has to be configured with the same cluster identifier) and set the "language" environment variable to the language you want to hear the output voice. If you do not know the language code, you can find it [here](https://www.andiamo.co.uk/resources/iso-language-codes/).
+Check in the YAML file that the cluster identifier is defined (OSCAR-CLI must be configured with the same cluster identifier). The "language" environment variable must be set to the language you want to hear the output voice in the Google speech library example. If you do not know the language code, it can be found [here](https://www.andiamo.co.uk/resources/iso-language-codes/).
 
-```
+``` yaml
 functions:
   oscar:
   - oscar-cluster:
-      name: text-to-speech
+      name: text-to-speech-google
       memory: 1Gi
       cpu: '1.0'
-      image: ghcr.io/grycap/text-to-speech
+      image: ghcr.io/grycap/text-to-speech-google
       script: script.sh
       log_level: CRITICAL
       input:
       - storage_provider: minio
-        path: text-to-speech/input
+        path: text-to-speech-google/input
       output:
       - storage_provider: minio
-        path: text-to-speech/output
+        path: text-to-speech-google/output
       environment: 
         Variables:
           language: en
 ```
 
-
-
+``` yaml
+functions:
+  oscar:
+  - oscar-cluster:
+      name: text-to-speech-coqui
+      memory: 2Gi
+      cpu: '4.0'
+      image: ghcr.io/grycap/text-to-speech-coqui
+      log_level: CRITICAL
+      script: script.sh
+      input:
+      - storage_provider: minio
+        path: text-to-speech-coqui/input
+      output:
+      - storage_provider: minio
+        path: text-to-speech-coqui/output
+```
 
 #### STEP 1.2: Use OSCAR-CLI to Deploy the Service
 
 To deploy the service, use the command:
-```sh
-oscar-cli apply tts.yaml
-```
-![03-oscar-apply.png](../../images/blog/post-text-to-speech/03-oscar-apply.png)
 
+``` sh
+oscar-cli apply $YAML_file
```

Comment:
I suggest indicating here directly the name of the files:

```oscar-cli apply text-to-speech-google.yaml```
```oscar-cli apply text-to-speech-google.yaml```

And removing the image capture down below.